packages/block-editor/src/components/iframe/index.js

```diff
@@ -130,9 +130,6 @@ function Iframe( {
 	] = useResizeObserver();
 
 	const setRef = useRefEffect( ( node ) => {
-		node._load = () => {
-			setIframeDocument( node.contentDocument );
-		};
```

Comment:
@ellatrix I'm curious why this extra `_load()` function was added? It seems like it should be doing the same thing as `onLoad()` below.

Reply:
This is not the same as [document load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) which only happens after the initial DOM is loaded and assets are loaded. The load function is called before the browser moves on to parse any other styles and script and the body tag below. It's needed because we're rendering our own body element, which (if I remember correctly) must be available to any scripts loaded (instead of the one we're removing). See #52405.